modules/apps/publisher/modules/ext/scripts/webapp.action.save.js

```diff
@@ -47,7 +47,7 @@ var module = function () {
 	}
 
     //TODO: Change this method to take WebAppObj as argument instead of passing properties separately.
-    function addToWebApp(uuid,webappProvider, webappName, webappVersion, webappContext, webappTrackingCode,asset, ssoEnabled, idpProviderUrl, saml2SsoIssuer, logoutURL) {
+    function addToWebApp(uuid,webappProvider, webappName, webappVersion, webappContext, webappTrackingCode,asset, ssoEnabled, idpProviderUrl, saml2SsoIssuer, logoutURL,allowAnonymous) {
```

Comment:
Added jira https://wso2.org/jira/browse/APPM-1072
This issue in repeated in many instance and all the scenarios will be fixed in future releases.